keyboards/mwstudio/cover65/info.json

```
@@ -0,0 +1,117 @@
+{
+    "manufacturer": "MWStudio",
+    "keyboard_name": "Cover65",
+    "maintainer": "TW59420",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+		"encoder": true,
+		"rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["A7", "B0", "B1", "B10", "B11", "B12", "B13", "B14", "B15", "A15", "B3", "B4", "B5", "B6", "B7" ],
+        "rows": ["A3", "A4", "A5", "A6", "A2" ]
+    },
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
```

This sets the processor, bootloader and board to f103+stm32duino
```suggestion
    "developmennt_board": "bluepill"
```